### PR TITLE
Remove v2 packages

### PR DESCRIPTION
**Pull request by daschult**
Removes the v2 packages and moves their contents down into the main packages.

### CODE REVIEW

File: client-runtime/src/main/java/com/microsoft/rest/interceptors/BaseUrlHandler.java [hunk 1]
```diff
@@ -1,53 +0,0 @@
...
-
-import java.io.IOException;
-
-/**
- * Handles dynamic replacements on base URL. The arguments must be in pairs
- * with the string in raw URL to replace as replacements[i] and the dynamic
- * part as replacements[i+1]. E.g. {subdomain}.microsoft.com can be set
- * dynamically by setting header x-ms-parameterized-host: "{subdomain}, azure"
- */
-public final class BaseUrlHandler implements Interceptor {
```
Comment (RikkiGibson) [inline client-runtime/src/main/java/com/microsoft/rest/interceptors/BaseUrlHandler.java]: This is now handled by RestProxy, right?

Comment (daschult) [reply client-runtime/src/main/java/com/microsoft/rest/interceptors/BaseUrlHandler.java]: Correct.

**Comment by jianghaolu** [comment]
We will need to keep v2 eventually - so that people can use v1 and v2 side by side.

**Comment by daschult** [comment]
What's the scenario that someone would use v1 and v2 side by side?

**Comment by RikkiGibson** [comment]
@daschult It's needed if someone is using both azure-sdk-for-java and, say, azure-keyvault-java or any of the other standalone SDKs that hasn't updated to the new runtime yet. It seems that other widely depended on packages like RxJava and OkHttp use a similar convention for dealing with this, which we've seen in all the `import okhttp3.something` import statements in our code.

**Pull request merged**